README.md

```diff
@@ -87,26 +87,27 @@ Before you install the plug-in, meet the following requirements:
 
 5.  [Create a user profile](#create-a-user-profile).
 
-## Create a User Profile
+## Creating a user profile
 
-You can create a `zftp` user profile to avoid typing your connection details on every command. A `zftp` profile contains the host, port, username, and password for the z/OS you will connect. You can create multiple profiles and switch between them as needed.
+After you install the plug-in, you create an FTP profile. An FTP profile is recommended to issue commands to via FTP. FTP profiles contain your host, port, user name, and password for connect to z/OS using FTP. You can create multiple profiles and switch between them as needed.
 
 **Follow these steps:**
-1.  Create a zftp profile:
+1.  Install the z/OS FTP Plug-in for Zowe CLI
+2.  Create an FTP profile:
 
     ```
-    zowe profiles create zftp <profile name> -H <host> -u <user> -p <password> -P <port> --secure-ftp true
+    zowe config init
     ```
+3.  If using a non-standard port, set the port number to your FTP connection:
 
-    The result of the command displays as a success or failure message. You can use your profile when you issue commands in the zftp command group.
-
-**Notice** The option `--secure-ftp true` is strongly recommended if FTPS (FTP over SSL) is enabled in z/OS FTP service. This is not the same as SFTP (FTP over SSH).
-
-**Tip:** For more information about the syntax, actions, and options, for a profiles create command, open Zowe CLI and issue the following command:
+    ```
+    zowe config set profiles.zftp.properties.port <port number>
+    ```
+4. If using a insecure connection, set the secureFtp value to false:
 
-```
-zowe profiles create zftp -h
-```
+    ```
+    zowe config set profiles.zftp.properties.secureFtp false
+    ```
 
 ## Running tests
```

Comment:
More notes:
Line 121: Not clear what the user needs to do once the file is copied. Should the user rename the copied file to `custom_properties.yaml`?
Line 123: change the tense: "... where the system test will be **ran** for the first time ... ." Also, data set it two words. Also: " ... **issue** the following command ..."
Line 127: Indent this content and split it up into separate lines, one for localfile and one for dataset
Line 139: lowercase: Uninstall the plug-in
Line 146: state what the actual "product" is
Line 162: we should explain why we're including this example script. It's not clear as it is now.

Reply:
> Line 162

Added some comments to the script and slightly updated the intro to the script 😋